js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -81,40 +81,42 @@
         <p><code>disableGrammar</code> Grammar features such as Syllables, Parts of speech, and Picture Dictionary will be disabled.</p>
         <p><code>disableLanguageDetection</code> The Language Detection feature will be disabled. This will prevent Immersive Reader from automatically detecting the language of the supplied content.</p>
         <p><code>disableTranslation</code> The Translation features will be disabled.</p>
-        <p><code>parent</code> Immersive reader is launched in the specified element, otherwise will be launched in the main document body.</p>
+        <p><code>parent</code> Immersive reader iframe is placed the specified element, otherwise it will be placed in the main document body.</p>
         <h3>Examples:</h3><br />
-        <section class='section' id='SampledisableLanguageDetection'>
-            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableLanguageDetection')"></div>
-            Specify the content language: <input id='langdisableLanguageDetection' value='en' /> (i.e 'fr', 'es', 'de', 'zh')
-            <h3 id='titledisableLanguageDetection'>Geography (Speech, Syllables, Parts of Speech and Picture Dictionary are all affected by the language.)</h3>
-            <p id='textdisableLanguageDetection'>The study of Earth's landforms is called physical geography. Landforms can be mountains and valleys. They can also be glaciers, lakes or rivers. Landforms are sometimes called physical features. It is important for students to know about the physical geography of Earth. The seasons, the atmosphere and all the natural processes of Earth affect where people are able to live. Geography is one of a combination of factors that people use to decide where they want to live.</p>
+        <section class='section' id='sampleDisableLanguageDetection'>
+            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('DisableLanguageDetection')"></div>
```

Comment:
camel case

disableLanguageDetection


for consistency

Reply:
I must keep in that way because JS code search for the elements ids in camel case , example= **text**DisableGrammar, **title**DisableTranslation.